install/kubernetes/helm/istio/charts/gateways/values.yaml

```diff
@@ -25,7 +25,7 @@ istio-ingressgateway:
   cpu:
     targetAverageUtilization: 80
   loadBalancerIP: ""
-  loadBalancerSourceRanges: {}
+  loadBalancerSourceRanges: []
```

Comment:
I think the comment is we should just use an empty value here.

Reply:
We were thinking an empty array helps the consumers of the Istio chart see the data format required. Is there any downside to specifying an empty array?

Reply:
Please check some discussion at https://github.com/istio/istio/pull/9258#pullrequestreview-163643788